tron/config/schema/tronfig_schema.json

```diff
@@ -0,0 +1,218 @@
+                  }
+                },
+                "nodes": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "hostname": {
+                          "type": "string"
+                        },
+                        "name": {
+                          "type": "string"
+                        },
+                        "username": {
+                          "type": "string"
+                        },
+                        "port": {
+                          "type": "number"
+                        }
+                      }
+                    }
+                },
+                "node_pools": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "nodes": {
+                          "type": "array",
+                          "items": {
+                            "type": "string"
+                          }
+                        }
+
+                    }
+                }
+                }
+                },
+              "jobs": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string"
+                      },
+                      "node": {
+                        "type": "string"
+                      },
+                      "schedule": {
+                        "oneOf" :  [
+                          {
+                            "type": "string"
+                          },
+                          {
+                            "type": "object"
+                          }
+                        ]
+                      },
+                      "actions": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "name":{
+                              "type": "string"
+                            },
+                            "command": {
+                              "type": "string"
+                            },
+                            "requires": {
+                              "type": "array",
+                              "items": {
+                                "type": "string"
+                              }
+                            },
+                            "node": {
+                              "type": "string"
+                            }
+                          }
+                        }
+                      },
+                      "queuing": {
+                        "type": "boolean",
+                        "default": "true"
```

Comment:
here and in other places: use json correct primitives for default values (actual boolean `false`, not string `"false"`)

Reply:
im not sure you can in jsonschema, but I'll double check

Reply:
if that's the case, then does this line needs fixing: https://github.com/Yelp/Tron/pull/334/files#diff-a7ea5d2cfec07683e11d09dec94d44edR14 and the other places where there are number literals?